extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/OpenshiftProcessorUtil.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.kubernetes.deployment;
+
+import static io.quarkus.kubernetes.deployment.Constants.OPENSHIFT_ROUTE_TARGET_PORT;
+
+import java.util.Optional;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+public class OpenshiftProcessorUtil {
+
+    public static Optional<String> getRouteTargetPort() {
+        return ConfigProvider.getConfig().getOptionalValue(OPENSHIFT_ROUTE_TARGET_PORT, String.class);
```

Comment:
Just as in https://github.com/quarkusio/quarkus/pull/27027, it would be nice to have this in a `@ConfigItem`

Reply:
I’ve made this change but also had a bit of a nightmare with git which has meant some of these changes shouldn’t still be here. Will resolve in the morning!